test/compilable/testCtfeWriteln.d

```diff
@@ -0,0 +1,31 @@
+/*
+REQUIRED ARGS: -c
+TEST OUTPUT:
+0^^2 == 0
+1^^2 == 1
+2^^2 == 4
+3^^2 == 9
+4^^2 == 16
+5^^2 == 25
+6^^2 == 36
+result == 91
+*/
+
+int sum_of_sq(int x) pure nothrow @safe
+{
+    import std.conv;
+    int result = 0;
+    foreach (i; 0 .. x)
+    {
+        __ctfeWrite(i.to!string, "^^2 == ");
+       int power = i ^^ 2;
+        __ctfeWriteln(power.to!string);
```

Comment:
Wait where is `__ctfeWriteln` coming from? I thought only `__ctfeWrite` was implemented?